docs/reference/run.md

```diff
@@ -773,7 +773,7 @@ capabilities using `--cap-add` and `--cap-drop`. By default, Docker has a defaul
 list of capabilities that are kept. The following table lists the Linux capability options which can be added or dropped.
 
 | Capability Key | Capability Description |
-| :----------------- | :---------------| :-------------------- |
+| -------------- | ---------------------- |
```

Comment:
I think this modification should not be there (as it is in another PR if I remember correctly).